os/include/tinyara/pm/pm.h

```diff
@@ -303,9 +303,15 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
-struct pm_wakeup_timer_s {
-	bool use_timer;
-	uint32_t timer_interval;
+enum pm_timer_type_e {
+	PM_WAKEUP_TIMER = 1,
+	PM_LOCK_TIMER = 2,
+	/* Scope for future expansion, up to 8 timer types can be supported */
+};
+
+struct pm_timer_s {
+	uint8_t timer_type;		/* Bits here are set according to the timer that is to be used */
```

Comment:
I assume bit are updated according to  pm_timer_type_e . Documentation can be updated accordingly.

Reply:
Bit masking is not supported currently as synchronous multi timer support is not a part of the scope right now.
That will come in next.